client/src/store/reducer.ts

```diff
@@ -6,7 +6,13 @@ The full terms of this copyright and license should always be found in the root
 */
 import { normalizeString } from "utils";
 import {
+  VISIBILITY_SINGLE_QUESTION,
+  visibilityAnswerAction,
```

Comment:
confusing to have these two things named so similarly. Are they both real/used?

Reply:
I changed their names, and both are used. One for `CHAT_QUESTION_VISIBILITY_SET` and the other for `CHAT_QUESTION_VISIBILITY_SHOW_ALL`